### ISSUE TITLE
require_once(.../vendor/autoload.php) error while enabling the module.

### ISSUE THREAD

**Issue by pathintegral** [mnylc/islandora_multi_importer#25]
Hi,

I cloned the repository to my modules directory then tried to enable it via `drush en -y islandora_multi_importer`. But an error came up with following content:

> require_once(/var/www/drupal7/sites/all/modules/islandora_multi_importer/vendor/autoload.php): failed to open stream: No such file or directory islandora_multi_importer.module:17
> 
> PHP Fatal error:  require_once(): Failed opening required '/var/www/drupal7/sites/all/modules/islandora_multi_importer/vendor/autoload.php' (include_path='.:/usr/share/php:/usr/share/pear') in /var/www/drupal7/sites/all/modules/islandora_multi_importer/islandora_multi_importer.module on line 17
> 
> Drush command terminated abnormally due to an unrecoverable error.                                                                                                                                     
> 
> Error: require_once(): Failed opening required '/var/www/drupal7/sites/all/modules/islandora_multi_importer/vendor/autoload.php' (include_path='.:/usr/share/php:/usr/share/pear') in
> /var/www/drupal7/sites/all/modules/islandora_multi_importer/islandora_multi_importer.module, line 17

Thanks.

Edit: I also tried to install on another Islandora server. It's enabled perfectly this time. When I try to reach website from web browser, i see following error instead of homepage:

> Fatal error: require_once(): Failed opening required '/var/www/drupal7/sites/all/modules/islandora_multi_importer/vendor/autoload.php' (include_path='.:/usr/share/php:/usr/share/pear') in /var/www/drupal7/sites/all/modules/islandora_multi_importer/islandora_multi_importer.module on line 17

**Comment by DiegoPino** [comment]
Hi, you need to run `composer update` inside the module's folder to download the required libraries and generate the class autoloader.

**Comment by DiegoPino** [comment]
Will reopen the issue and add instructions to the README so this gets documented for people new to composer

**Comment by pathintegral** [comment]
Thanks for reply. 
I wasn't familiar with composer but figured it out, then. It would be great that add instructions to README.

**Comment by DiegoPino** [comment]
Cool, will do today. Please feel free to make any questions regarding the use of this module in Islandora. We use it in production a lot and we know of a few other institutions that have been using it too, but it can require some testing before getting used to it. I will be adding a lot of improvements in the next weeks, so stay tuned if you happen to like it. Thanks!

**Comment by rosiel** [comment]
Yep, just ran into this too. :) i hope `composer install ` is ok...

**Comment by mandalalifeart** [comment]
i made compose json in my app root folder on my pc , I ran composer update from gitbash , deployed both json and lock  file , but looks like vendor folder was not created :
 PHP Fatal error:  require_once(): Failed opening required '/app/www/vendor/autoload.php'

**Comment by DiegoPino** [comment]
@mandalalifeart  You need to run `composer install` from inside this drupal's module folder.  Why app?
